python/ingestion/cdc_wonder_utils.py

```diff
@@ -38,7 +38,7 @@
     # Age source groups already match needed HET groups
     std_col.RACE_CATEGORY_ID_COL: {
         'American Indian or Alaska Native': std_col.Race.AIAN_NH.value,
-        'Asian or Pacific Islander': std_col.Race.API_NH.value,
+        'Asian or Pacific Islander': std_col.Race.ASIAN_PAC_NH.value,

```

Comment:
API is the one aligned with census/cdc races; ASIAN_PAC is a weird one off only used by CAWP because they lump native hawaiian elsewhere

Reply:
Ah okay! Thanks!